stop_and_wait_plugin/src/stop_and_wait_plugin.cpp

```diff
@@ -237,6 +237,10 @@ std::vector<cav_msgs::TrajectoryPlanPoint> StopandWait::compose_trajectory_from_
 
   if (req_dist > remaining_distance)
   {
+    while (remaining_distance <= 0.0 && remaining_distance <= (stop_location - starting_downtrack)){
+      remaining_distance += 0.2;
+    }
```

Comment:
See comment 
https://github.com/usdot-fhwa-stol/carma-platform/pull/1573#discussion_r777483159

Reply:
This also looks like its missing the comments requested in the linked PR. Again it seems like this branch is missing that change set.

Reply:
never mind, github caching didn't show this correctly on my end.